client/src/routes/Session/hooks/useLeaveSession.ts

```diff
@@ -1,28 +1,21 @@
 import {useCallback, useContext} from 'react';
 import {Alert} from 'react-native';
 import {useTranslation} from 'react-i18next';
-import {useNavigation} from '@react-navigation/native';
-import {NativeStackNavigationProp} from '@react-navigation/native-stack';
 import {useResetRecoilState} from 'recoil';
 
 import {DailyContext} from '../DailyProvider';
 import {sessionAtom} from '../state/state';
-import {TabNavigatorProps} from '../../../lib/navigation/constants/routes';
-
-type ScreenNavigationProps = NativeStackNavigationProp<TabNavigatorProps>;
 
 const useLeaveSession = () => {
   const {t} = useTranslation('Component.ConfirmExitSession');
   const {leaveMeeting} = useContext(DailyContext);
-  const {navigate} = useNavigation<ScreenNavigationProps>();
 
   const resetSession = useResetRecoilState(sessionAtom);
 
   const leaveSession = useCallback(async () => {
     await leaveMeeting();
     resetSession();
-    navigate('Sessions');
```

Comment:
This breaks exiting the session. It needs to navigate here since we use this hook for back buttons/behaviour.